### ISSUE TITLE
Accessibility errors found in your template

### ISSUE THREAD

**Issue by wapyce** [sergeyklay/sergeyklay.github.io#1]
The user @carlsonsantana validate your site template "https://sergeyklay.github.io/" and found these accessibility errors:
* Page https://sergeyklay.github.io/:
  * Issue *a5826b59-48f3-4826-a52d-013a8d4cb803*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_3.1_3_1.H49.AlignAttr
    * **Message**: Align attributes .
    * **Selector**: `#footer > p > img`
    * **Context**: `<img class="emoji" title=":heart:" alt=":heart:" src="https://assets-cdn.github.com/images/icons/emoji/unicode/2764.png" height="20" width="20" align="absmiddle">`
  * Issue *af102474-a258-4c3f-b562-4980966fd45b*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle4.Guideline4_1.4_1_1.F77
    * **Message**: Duplicate id attribute value "wrapper" found on the web page.
    * **Selector**: `#wrapper`
    * **Context**: `<section id="wrapper" class="home">
    <ul id="post-list">
    
 ...</section>`
  * Issue *ef90db48-95ea-4fe7-8421-070346216a8b*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_1.1_1_1.H30.2
    * **Message**: Img element is the only content of the link, but is missing alt text. The alt text should describe the purpose of the link.
    * **Selector**: `#header > a`
    * **Context**: `<a href="/about">
                <img id="avata...</a>`
  * Issue *1c5eb44a-f203-43b8-aa44-acd111b34b86*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
    * **Message**: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 2.42:1. Recommendation: change background to #287abd.
    * **Selector**: `html > body > nav > a:nth-child(2)`
    * **Context**: `<a class="cta" href="/feed.xml">Subscribe</a>`
  * Issue *a783123f-4266-4bb2-bfbf-91b8f7f6bebc*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
    * **Message**: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 2.42:1. Recommendation: change text colour to #287abd.
    * **Selector**: `html > body > nav > a:nth-child(1)`
    * **Context**: `<a href="/about">About </a>`

You can check these accessibility errors using [pa11y](https://github.com/pa11y/pa11y).
You can view the [full validation results in our website](https://wapyce.herokuapp.com/validation/validations/02b90e8a-9071-41fb-af91-49fa627caca4/).
